generators/entity-client/templates/react/src/main/webapp/app/entities/entity-dialog.tsx.ejs

```diff
@@ -87,7 +107,18 @@ import { getEntities as get<%= upperFirstCamelCase(uniqueRel.otherEntityNamePlur
 <%_ }
   }
 }); _%>
-import { getEntity, updateEntity, createEntity } from './<%= entityFileName %>.reducer';
+import {
+  getEntity,
+  updateEntity,
+  createEntity,
+  <%_
+    blobs.forEach(b => {
+      const capitalizedFirstLetter =  b.fieldName.charAt(0).toUpperCase() + b.fieldName.slice(1);
```

Comment:
use lodash, please. If required create a method in generator-base-private and use that here, within the method use lodash to do this. something with `upperFirst`